docs/reference/authentication.rst

```diff
@@ -1,8 +1,9 @@
-.. _authenticate:
+.. _api-authentication:
+
+==================
+API Authentication
+==================
 
-==============
-Authentication
-==============
 
 The Flocker REST API uses TLS to secure and authenticate requests.
```

Comment:
I don't think we need a glossary per se, but we are going to call the local storage backend and all of the Labs projects 'something'.  The current recommendation based on a conversation with Mark is "experimental".  I'd like a page in docs that defines what that means.  There is a thread of Zulip asking Jean-Paul if he is over moving to experimental/stable instead of the alpha/beta/prod that we talked about before.